tests/suites/test_suite_mpi.function

```diff
@@ -890,20 +890,22 @@ void mpi_mod_raw_cond_assign( data_t * input_X,
     #define MAX_LEN 64
     mbedtls_mpi_uint *X = NULL;
     mbedtls_mpi_uint *Y = NULL;
-    mbedtls_mpi_uint buff_m[MAX_LEN];
+    mbedtls_mpi_uint *buff_m = NULL;
     mbedtls_mpi_mod_modulus m;
     size_t limbs_X = CHARS_TO_LIMBS( input_X->len );
     size_t limbs_Y = CHARS_TO_LIMBS( input_Y->len );
     size_t len_X = limbs_X * sizeof( mbedtls_mpi_uint );
     size_t len_Y = limbs_Y * sizeof( mbedtls_mpi_uint );
+    size_t len_m = len_Y;
 
     TEST_ASSERT( limbs_X <= MAX_LEN );
     TEST_ASSERT( limbs_Y <= MAX_LEN );
 
     ASSERT_ALLOC( X, len_X );
     ASSERT_ALLOC( Y, len_Y );
 
-    memset( buff_m, 0, MAX_LEN * sizeof( mbedtls_mpi_uint ) );
+    ASSERT_ALLOC( buff_m, len_m );
+    memset( buff_m, 0, len_m );
```

Comment:
Non-blocker: `ASSERT_ALLOC` sets the buffer to 0 (we never allocate without setting to zero), so this memset is redundant. Same in other places below.